.github/workflows/tox.yml

```diff
@@ -85,3 +85,38 @@ jobs:
           if [[ "$TOXENV" == 'csslint' ]]; then npm install -g csslint; fi
       - name: Run tox
         run: python -m tox
+
+  translation:
+    strategy:
+      fail-fast: false
+      max-parallel: 4
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Install Python dependencies
+        run: |
+          python -m pip install -r requirements/project.txt
+      - name: Check if Portable Object Templates should have been updated
+        run: |
+          python setup.py extract_messages -t mkdocs
+          python setup.py extract_messages -t readthedocs
+          git diff -G "msgid" --exit-code mkdocs/themes/*/*.pot
+        shell: bash
+      - id: files
+        uses: jitterbit/get-changed-files@v1
+      - name: Check if Portable Object translation files should be updated on mkdocs version change
+        run: |
+          for locale in $(ls mkdocs/themes/mkdocs/locales/); do python setup.py update_catalog -t mkdocs -l ${locale}; done
+          for locale in $(ls mkdocs/themes/readthedocs/locales/); do python setup.py update_catalog -t readthedocs -l ${locale}; done
+          for changed_file in ${{ steps.files.outputs.all }}; do
+            echo "inspecting changed file: ${changed_file}";
+            if [[ "${changed_file}" == "mkdocs/__init__.py" ]]; then
+              echo "mkdocs version changed!";
+              git diff -G "msgid" --exit-code && git diff -G "msgstr" --exit-code;
+            fi
+          done
```

Comment:
I have taken a close look at the workflow additions and they look good overall, so long as they work as expected.

I do have a couple thoughts about this specific loop though:

1. I don't think we need the `echo` statement in line 116 (`echo "inspecting changed file: ${changed_file}";`). That will just muddy up the logs with a list of every changes file in the PR. In understand why you might have included it during development for debugging purposes, but if we have a working run, then we can remove it.
2. Is there a reason why you didn't include a `break` statement in the `if` after running `git diff`? Once we have confirmed `mkdocs/__init__.py` is in the list of changed files, we don't need to check any other files. Or does the break statement obviate the exit code from `git diff`?

Reply:
I see you have a sample run which intentionally fails [here](https://github.com/ultrabug/mkdocs/runs/2169842544?check_suite_focus=true) and one that passes [here](https://github.com/ultrabug/mkdocs/runs/2169835768?check_suite_focus=true). And then [here}(https://github.com/ultrabug/mkdocs/runs/2169577810?check_suite_focus=true) is one which fails when checking POT files. They all look good, Thanks.

I think that addresses all of the concerns I had.